### PR TITLE
Add HCA Chapter 3

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2234

The new is on the left, the old is on the right.
image

Note: The page title doesn't show up...not sure why, but this is a ubiquitous issue, so I'm not worrying about it in this PR.

### CODE REVIEW

**Comment by jbalboni** [comment]
@cvalarida If you want the page title to show up there, it should go in `ui:title` in the root of `uiSchema`. The other title is used on the review page, but doesn't automatically show up on the regular pages because we don't always want a title to show up.

File: src/js/hca-rjsf/config/form.js [hunk 1]
```diff
@@ -233,8 +236,40 @@ const formConfig = {
         }
       }
     },
+    vaBenefits: {
+      title: 'VA Benefits',
+      pages: {
+        vaBenefits: {
+          path: 'va-benefits/basic-information',
+          title: 'Current Compensation',
```
Comment (jbalboni) [inline src/js/hca-rjsf/config/form.js]: Page titles should be sentence case. Chapter titles are title case, so thank you for updating those.

**Review by jbalboni — APPROVED** [review]
LGTM, but needs some tests. Also, should be able to add the page title based on my comment.

File: src/js/hca-rjsf/config/form.js (hunk 1 above)
Comment (cvalarida) [inline src/js/hca-rjsf/config/form.js]: Oh blast. You're right. I just copied and pasted this from the old one without thinking. Good catch!

**Pull request merged**